src/providers/be_refresh.c

```diff
@@ -136,16 +136,16 @@ struct be_refresh_ctx {
     struct be_refresh_cb callbacks[BE_REFRESH_TYPE_SENTINEL];
```

Comment:
Just a comment about the commit message. I think it is now even less odd. It helps to hide details of be_ctx. But mainly because the old usage of `be_ctx->refresh_ctx` when calling `be_ptask_create()` was really odd.